test/triage_test.js

```diff
@@ -0,0 +1,130 @@
+const mocha = require('mocha');
+const describe = mocha.describe;
+const it = mocha.it;
+const proxyquire = require('proxyquire').noCallThru();
+const sinon = require('sinon');
+const assert = require('assert');
+
+let functions, autoMlMock, octoMock, dataBuffer, mockSettings;
+
+beforeEach(() => {
+  const data = JSON.stringify({
+    owner: 'GoogleCloudPlatform',
+    repo: 'labelcat',
+    number: 22,
+    text: 'some issue information',
+  });
+
+  dataBuffer = Buffer.from(data);
+
+  const reject = sinon.stub();
+  reject.withArgs(true).throws(401);
+
+  const mockAdd = sinon.stub().returns(
+    Promise.resolve({
+      data: [
+        {
+          id: 271022241,
+          node_id: 'MDU6TGFiZWwyNzEwMjIyNDE=',
+          url:
+            'https://api.github.com/repos/GoogleCloudPlatform/LabelCat/labels/bug',
+          name: 'bug',
+          color: 'fc2929',
+          default: true,
+        },
+      ],
+      status: 200,
+    })
+  );
+
+  octoMock = {
+    authenticate: sinon.stub(),
+    issues: {addLabels: mockAdd},
+  };
+
+  const model = sinon.spy();
+  const predict = sinon.stub();
+
+  predict.onCall(0).returns([
+    {
+      payload: [
```

Comment:
Assuming `classification` is the only thing that differs, can you put this return value in a function? Perhaps like the code below:
```javascript
const makePayload = (classification, displayName) => {
  return {
    annotationSpecId: '',
    displayName: displayName,
    classification: classification,
    detail: 'classification'
  }
}

predict.onCall(0).returns([
  payload: [makePayload([Object], 0), makePayload({score: 90}, 1)]
]);
...
```